### ISSUE TITLE
mWeb - Xero - 2FA animation slides in and out slowly

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#51254]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.52-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Issue was found when executing this PR:** https://github.com/Expensify/App/pull/50790
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Log in with a Gmail account without 2FA
2. Create a workspace
3. Navigate to Workspace settings - More features
4. Enable "Accounting"
5. Navigate to Workspace settings - Accounting
6. Tap on the "Connect" button next to Xero
7. Dismiss the modal

## Expected Result:
The animation should be smooth.

## Actual Result:
2FA animation slides in and out slowly.

## Workaround:
Unknown

## Platforms:

- [ ] Android: Standalone
- [ ] Android: HybridApp
- [x] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [x] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/f583417c-dd41-4525-b067-fc605a096281

 </details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by IuliiaHerets** [comment]
We think that this bug might be related to #wave-collect - Release 2

**Comment by IuliiaHerets** [comment]
@adelekennedy  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by zfurtak** [comment]
I tested the cases also without the changes made in [my PR](https://github.com/Expensify/App/pull/50790) and here is the comparison:

Firstly, I think the problem is only related to IOS mweb, on Android it looks correct (in my opinion).

<details>

<summary>Comparison on Android</summary>


https://github.com/user-attachments/assets/c4d8c4bc-32ad-4a04-9eef-b027bb67fc74 

https://github.com/user-attachments/assets/9c33e1c2-5f13-4e3b-8d37-c0628537fc73


</details>

Secondly - IOS mweb, the slow animation is there even without my modified code, it already looked rather rough. For comparison, I checked in debug mode, but I also looked at the app in production and there's not much difference. Below you can see the behaviour before my changes and after.

It is indeed a bug, but not caused by my code, at most it might be more visible because my code adds a sliding out animation that was not there before.

<details>

<summary>Comparison on IOS</summary>


https://github.com/user-attachments/assets/fe1ea38c-03fd-448d-bd13-26c7ca6fa466


https://github.com/user-attachments/assets/826efe74-ba18-4bfb-b28d-fb76f6ba6d6e


</details>

However, I can address this bug and try to fix the rough animation as it is a part of [our issue to improve the UX](https://github.com/Expensify/App/issues/49354).

**Comment by situchan** [comment]
yup, this is not a regression.

Also QA team stated that **Reproducible in production?: Y**

**Comment by adelekennedy** [comment]
@rushatgabhane thoughts on the above?

**Comment by adelekennedy** [comment]
I can't reproduce this so I think we've resolved here - closing!

**Comment by mkzie2** [comment]
@adelekennedy Issue is reproducible https://github.com/Expensify/App/pull/52232#issuecomment-2473146073. Please reopen.

cc @zfurtak Can you retest?

**Comment by zfurtak** [comment]
yeah, I think it is still visible, the modal animation is not so smooth

https://github.com/user-attachments/assets/570b67c4-24d7-4202-8256-9e6bbfb8eae9
